src/app/service/game-control.service.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable, filter, map, scan } from 'rxjs';
+
+export type move = {
+  field: number;
+  player: string;
+};
+@Injectable({
+  providedIn: 'root',
+})
+export class GameControlService {
+  private gameBoard: BehaviorSubject<string[]> = new BehaviorSubject<string[]>(
+    new Array(9)
+  );
+  private moves: BehaviorSubject<move> = new BehaviorSubject<move>({} as move);
```

Comment:
Let's do Subject?

Reply:
changed to games$$ BehaviorSubject with ReplaySubject as value